src/test/java/pageFactory/baseEntities/BasePage.java

```diff
@@ -51,14 +51,22 @@ protected void waitForOpen() {
             throw new RuntimeException("Page was not opened");
         }
     }
-    public void screenshot(){
-        File screenshot = ((TakesScreenshot)driver).getScreenshotAs(OutputType.FILE);
+
+    public void screenshot() {
+        byte[] screenshot = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
         Date dateNow = new Date();
-        SimpleDateFormat format = new SimpleDateFormat("d MMM yyyy hh-mm-ss");
+        SimpleDateFormat format = new SimpleDateFormat(DATEFORMAT);
         String fileName = format.format(dateNow) + ".png";
 
+        FileOutputStream file;
+        try {
+            file = new FileOutputStream("D://Screenshots//" + fileName);
```

Comment:
also an issue, do not use absolute paths in the future, try smthing like ./screenshots